setup.py

```diff
@@ -65,9 +65,10 @@
     ]
 
 testing_extras = tests_require + [
+    'mock',
```

Comment:
By the way this dependencies are only installed when you run tests which is not something you do when you use the project.